### PR TITLE
[System-Probe] remove code duplication in modules config pkgs

### PR DESCRIPTION
**Pull request by val06**
### What does this PR do?

Switched to use systemprobe/config.FullKeyPath to construct a full key path in the config from different fragments

### Motivation

remove code duplication

### CODE REVIEW

File: pkg/eventmonitor/config/config.go [hunk 1]
```diff
@@ -34,21 +33,17 @@ type Config struct {
...
 }
 
-func join(pieces ...string) string {
-	return strings.Join(pieces, ".")
-}
-
 func getAllKeys(key string) (string, string) {
-	deprecatedKey := strings.Join([]string{rsNS, key}, ".")
-	newKey := strings.Join([]string{evNS, key}, ".")
+	deprecatedKey := sysconfig.FullKeyPath(rsNS, key, ".")
```
Comment (paulcacheux) [inline pkg/eventmonitor/config/config.go:45]: are you sure about this ? I'm surprised we are keeping the "." here

**Review by paulcacheux — CHANGES_REQUESTED** [review]

File: pkg/eventmonitor/config/config.go (hunk 1 above)
Comment (paulcacheux) [reply pkg/eventmonitor/config/config.go:45]: No, my main issue is that
```
strings.Join([]string{rsNS, key}, ".") = `{rsNS}.{key}`
```
while
```
sysconfig.FullKeyPath(rsNS, key, ".") =  `{rsNS}.{key}..`
```

so I think it should just be `sysconfig.FullKeyPath(rsNS, key)` without the dot as the last argument

Comment (val06) [reply pkg/eventmonitor/config/config.go:45]: ahh lol, yeah my bad

**Review by paulcacheux — APPROVED** [review]

**Review by usamasaqib — APPROVED** [review]

**Review by akarpz — APPROVED** [review]

**Pull request merged**